Sources/DateScrubber.swift

```diff
@@ -69,20 +81,48 @@ public class DateScrubber: UIViewController {
         }
     }
 
+    private var sectionLabelState = State.Inactive {
+        didSet {
+
+            if self.sectionLabelState == .Active {self.setSectionLabelActive()}
+            if self.sectionLabelState == .Inactive {self.setSectionLabelInactive()}
+            self.updateSectionLabelFrame()
+        }
+    }
+
+    private var scrubberState = State.Inactive {
+        didSet {
+            self.updateDateScrubberFrame()
+        }
+    }
+
```

Comment:
🔝 here you see how I use the enum for the sectionLabelState and the scrubberState